OHHTTPStubs/OHHTTPStubs.xcodeproj/project.pbxproj

```diff
@@ -1735,6 +1746,7 @@
 				SDKROOT = appletvos;
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+				SWIFT_VERSION = 4.2;
```

Comment:
That means that the library itself is compatible with Swift 4.2 👍 but as a result it could be worth updating the `OHHTTPStubs.podspec` to add that `s.swift_version` to the list 😉

Reply:
Mmmh I see we actually never added that `s.swift_version` line to the podspec actually… might be the occasion to do so then

http://blog.cocoapods.org/CocoaPods-1.4.0/#swift-version-dsl